xss/xss.go

```diff
@@ -28,6 +28,26 @@ func queryHandler(w http.ResponseWriter, r *http.Request, safety string) (templa
 
 }
 
+// bufferedQueryHandler used as a handler which uses bytes.Buffer for source input ignoring the user input
+func bufferedQueryHandler(w http.ResponseWriter, r *http.Request, safety string) (template.HTML, bool) {
+	var buf bytes.Buffer
+	buf.Write([]byte("<script>"))
+	buf.WriteString("alert('buffered input - ")
+	buf.WriteRune('©')
+	buf.WriteString("');")
+	buf.WriteString("</script")
+	buf.WriteByte(byte('>'))
+
+	input := string(buf.Bytes())
+
+	if safety == "safe" {
```

Comment:
I noticed that when we check safety (eg unsafe,  safe, and noop), we switch between if/else and switch case statements.
I think sticking to just one method of doing this check would be good, to maintain some form of consistency as we make updates to this test app.

Reply:
Agreed about the consistency. I'll update `xss.go` and `unvalidated-redirect.go`